### PR TITLE
Store logits and recreate classifier when class is cleared

### PR DESCRIPTION
**Pull request by microbit-grace**
Recreating classifier is a way around breaking inference when certain classes are cleared

https://classifier-fix.teachable-machine-v1.pages.dev/

Tested on Chromebook and seems to work ok!

### CODE REVIEW

File: src/ai/WebcamClassifier.js [hunk 1]
```diff
@@ -167,17 +169,24 @@ export default class WebcamClassifier {
     const img = tf.fromPixels(image);
     const logits = this.mobilenetModule.infer(img, 'conv_preds');
     this.classifier.addExample(logits, newMappedIndex);
+    this.mappedClassifierLogits[newMappedIndex] =
```
Comment (microbit-matt-hillsdon) [inline src/ai/WebcamClassifier.js]: Maybe just:

(this.mappedClassifierLogits[newMappedIndex] ??= []).push(logits)

There's no reason not to mutate the array here I think.

**Review by microbit-matt-hillsdon — APPROVED** [review]

File: src/ai/WebcamClassifier.js (hunk 1 above)
Comment (microbit-grace) [reply src/ai/WebcamClassifier.js]: Tweaked it [7751a88](https://github.com/microbit-matt-hillsdon/teachable-machine-v1/pull/13/commits/7751a88c92a3703227c2dabe706b80e20ade33dd)
